website/docs/d/region.html.markdown

```diff
@@ -35,12 +33,9 @@ exported as attributes.
 * `name` - (Optional) The full name of the region to select.
 
 * `current` - (Optional) Set to `true` to match only the region configured
-  in the provider. (It is not meaningful to set this to `false`.)
+  in the provider. Defaults to `true` if `endpoint` or `name` is not given.
```

Comment:
I'm curious what is the value of `current = true` still. 🤔  Can't we just assume it's always true in such conditions and deprecate the field?